addons/web/static/src/views/fields/boolean/boolean_field.xml

```diff
@@ -2,7 +2,7 @@
 <templates xml:space="preserve">
 
     <t t-name="web.BooleanField" owl="1">
-        <CheckBox id="props.id" value="props.value or false" disabled="isReadonly" onChange.bind="onChange" />
+        <CheckBox id="props.id" value="props.value or false" className="'d-inline-block me-2'" disabled="isReadonly" onChange.bind="onChange" />
```

Comment:
There also used to be margin on the left of the checkbox, can you use mx-2 instead? Thanks